### PR TITLE
KILL commands docs added.

### PR DESCRIPTION
**Pull request by nizhikov**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: docs/_docs/sql-reference/operational-commands.adoc [hunk 1]
```diff
@@ -113,3 +113,120 @@ After executing the above command and *closing the JDBC connection*, all data wi
...
+----
+
+=== Description
+`ASYNC` is an optional parameter which returns control immediately without waiting for the cancellation to finish, and `query_id` can be retrieved via the `SQL_QUERIES` view.
+
+When a query cancelled via the `KILL` command, all parts of the query running on all other nodes are cancelled.
+
+This command available via JMX and CMD, also:
+
+- `QueryMXBean#cancelSQL`
```
Comment (dmagda) [inline docs/_docs/sql-reference/operational-commands.adoc:132]: Could you please wrap this command example (and all others down below) up with the ascii [tabs] tag? Where the first tab shows the snipped for Unix systems (control.sh) while the second is dedicated for Windows users (control.bat)?

Here is an example:
https://github.com/apache/ignite/blame/IGNITE-7595/docs/_docs/includes/starting-node.adoc#L10

File: docs/_docs/sql-reference/operational-commands.adoc [hunk 2]
```diff
@@ -113,3 +113,120 @@ After executing the above command and *closing the JDBC connection*, all data wi
...
+- `./control.sh --kill SQL query_id`
```
Comment (dmagda) [inline docs/_docs/sql-reference/operational-commands.adoc:133]: I would add a paragraph saying how to find the "query_id" parameter.

File: docs/_docs/sql-reference/operational-commands.adoc [hunk 3]
```diff
@@ -113,3 +113,120 @@ After executing the above command and *closing the JDBC connection*, all data wi
...
+KILL TRANSACTION 'xid'
+----
+
+=== Description
+`xid` of the transaction can be retrived via the `TRANSACTIONS` view.
+
+This command available via JMX and CMD, also:
+
+- `TransactionMXBean#cancel`
+- `./control.sh --kill TRANSACTION xid`
```
Comment (dmagda) [inline docs/_docs/sql-reference/operational-commands.adoc:150]: I would add an explanation on how to find the "xid" that needs to be passed into this command.

File: docs/_docs/sql-reference/operational-commands.adoc [hunk 4]
```diff
@@ -113,3 +113,120 @@ After executing the above command and *closing the JDBC connection*, all data wi
...
+KILL COMPUTE 'session_id'
+----
+
+=== Description
+`session_id` parameter can be retrived via the `TASKS` or `JOBS` view.
+
+This command available via JMX and CMD, also:
+
+- `ComputeMXBean#cancel`
+- `./control.sh --kill COMPUTE session_id`
```
Comment (dmagda) [inline docs/_docs/sql-reference/operational-commands.adoc:191]: Same here. Let's explain how to find the "session_id" and the "routine_id" for the continuous query-specific command below.

File: docs/_docs/sql-reference/operational-commands.adoc (hunk 3 above)
Comment (nizhikov) [reply docs/_docs/sql-reference/operational-commands.adoc:150]: I explained it few lines above:

`xid` of the transaction can be retrived via the `TRANSACTIONS` view.

File: docs/_docs/sql-reference/operational-commands.adoc (hunk 4 above)
Comment (nizhikov) [reply docs/_docs/sql-reference/operational-commands.adoc:191]: These parameters are the same as in SQL command.
And I explained how to find it in the system views.

File: docs/_docs/sql-reference/operational-commands.adoc (hunk 1 above)
Comment (nizhikov) [reply docs/_docs/sql-reference/operational-commands.adoc]: Fixed.

**Pull request merged**